rules/evasion/bitwise_math.yara

```diff
@@ -18,7 +19,63 @@ rule excessive_bitwise_math : high {
     hash_2023_yvper_0_1_setup = "b765244c1f8a11ee73d1e74927b8ad61718a65949e0b8d8cbc04e5d84dccaf96"
     hash_2023_aiohttpp_0_1_setup = "cfa4137756f7e8243e7c7edc7cb0b431a2f4c9fa401f2570f1b960dbc86ca7c6"
   strings:
-    $x = /\-{0,1}\d{1,8} \<\< \-{0,1}\d{1,8}/
+    $x = /\-{0,1}[\da-z]{1,8} \<\< \-{0,1}\d{1,8}/
   condition:
     filesize < 128000 and #x > 20
 }
+
+rule bitwise_math : low {
+  meta:
+    description = "uses bitwise math"
+  strings:
+    $x = /\-{0,1}[\da-z]{1,8} \<\< \-{0,1}\d{1,8}/
+    $y = /\-{0,1}[\da-z]{1,8} \>\> \-{0,1}\d{1,8}/
+  condition:
+    filesize < 65535 and any of them
+}
+
+rule bidirectional_bitwise_math : medium {
+  meta:
+    description = "uses bitwise math in both directions"
+	ref = "https://www.reversinglabs.com/blog/python-downloader-highlights-noise-problem-in-open-source-threat-detection"
+  strings:
+    $x = /\-{0,1}[\da-z]{1,8} \<\< \-{0,1}\d{1,8}/
+    $y = /\-{0,1}[\da-z]{1,8} \>\> \-{0,1}\d{1,8}/
+  condition:
+    filesize < 65535 and all of them
```

Comment:
Deleted my comment re: the difference between this and `bitwise_math` (misread the conditions).